docs: More Book tweaks for 0.4.0 (#1557)
PrQL/prql: book/src/integrations/README.md

# Integrations

> PRQL is building integrations with the external tools at the left, including:

Reviewer: ```suggestion
PRQL is building integrations with lots of external tools, including:
```
